app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dicecloud",
-  "version": "2.0.43",
+  "version": "2.0.44",
```

Comment:
Don't bump version number in pull requests, it gets bumped on release, which might include many changes and merged pulls